src/App.scss

```diff
@@ -29,6 +29,10 @@
   }
 }
 
+.select-style {
```

Comment:
I do understand why this exists but i am not sure if this is the way forward (right now there is no other option...). This is problem with forms inside modals. I will bring this UP with Susan and try to figure out some future proof solution.

Reply:
Sounds good, thank you!

Reply:
@Hyperkid123 - my next PR will replace the select with a compact table - that is in the UX mocks